x/genutil/client/cli/migrate.go

```diff
@@ -74,9 +73,17 @@ $ %s migrate v0.36 /path/to/genesis.json --chain-id=cosmoshub-3 --genesis-time=2
 			target := args[0]
 			importGenesis := args[1]
 
-			genDoc, err := tmtypes.GenesisDocFromFile(importGenesis)
+			genDoc, err := validateGenDoc(importGenesis)
 			if err != nil {
-				return errors.Wrapf(err, "failed to read genesis document from file %s", importGenesis)
+				return err
+			}
+
+			// Since some default values are valid values, we just print to
+			// make sure the user didn't forget to update these values.
+			if genDoc.ConsensusParams.Evidence.MaxBytes == 0 {
+				fmt.Printf("Warning: consensus_params.evidence.max_bytes is set to 0. If this is"+
+					" deliberate, feel free to ignore this warning. If not, please have a look at the chain"+
+					" upgrade guide at %s.\n", chainUpgradeGuide)
```

Comment:
i went through [this function](https://github.com/tendermint/tendermint/blob/e3b96107af324fc6a7761563c4cf58d6c4119ad6/types/genesis.go#L70) (as well as ValidateConsensusParams), and it seems to me that MaxBytes is the only default value that the user might have forgotten to migrate.

lmk if you can find other easy-to-forget values.